web/libs/editor/src/stores/Annotation/Annotation.js

```diff
@@ -194,11 +194,29 @@ const _Annotation = types
       user = user.id;
     }
 
+    const getCreatedBy = (snapshot) => {
+      if (snapshot.type === "prediction") {
+        const modelVersion = snapshot.model_version?.trim() ?? "";
+        return modelVersion || "Admin";
+      }
+
+      return snapshot.createdBy ?? "Admin";
+    };
+
+    const getCreatedAt = (snapshot) => {
+      if (snapshot.type === "prediction") {
+        return snapshot.created_at ?? snapshot.createdDate;
+      }
+      return snapshot.draft_created_at ?? snapshot.created_at ?? snapshot.createdDate;
```

Comment:
it could be simplified to 
```suggestion
      return snapshot.draft_created_at ?? snapshot.created_at ?? snapshot.createdDate;
```